plugins/inputs/pulsar_consumer/README.md

```diff
@@ -0,0 +1,9 @@
+# Pulsar Consumer Input Plugin
+
+Telegraf plugin to receive metrics from Apache Pulsar.
+
+### Configuration:
+
+```toml
+
```

Comment:
Make sure this matches the output of `SampleConfig()`, with the addition of `[[inputs.pulsar]]`